go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	go.viam.com/rdk v0.40.0
 	go.viam.com/test v1.1.1-0.20220913152726-5da9916c08a2
 	go.viam.com/utils v0.1.98
+	golang.org/x/exp v0.0.0-20240103183307-be819d1f06fc
```

Comment:
why this change?

Reply:
Forgot to run go mod tidy from last PR.

But thanks for pointing this out, for some reason I was using the external package instead of math/rand.

Reply:
Removed that dep from example script.